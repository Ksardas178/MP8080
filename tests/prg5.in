MVI B b01	//B <- 1
MVI A b11	//A <- 3
XRA B		//A <- b00000010
RAL			//A<<1
LXI SP 7 8	//SP = 00001000|00000111
PUSH PSW	//PSW -> stack
HLT
//PSW = accum|flags (S,Z,0,AC,0,P,1,C)

