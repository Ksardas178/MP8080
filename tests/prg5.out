
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: 00000110
00000001: 00000001
00000010: 00111110
00000011: 00000011
00000100: 10101000
00000101: 00010111
00000110: 00110001
00000111: 00000111
00001000: 00001000
00001001: 11110101
00001010: 01110110
